src/translations/nl.json

```diff
@@ -0,0 +1,291 @@
+{
+  "$schema": "./translation-schema.json",
+  "authentication": {
+    "account": "Account",
+    "accountOfCurrentUser": "Account van huidige gebruiker",
+    "alreadyActivated": "Al geactiveerd",
+    "alreadyLoggedIn": "Al ingelogd",
+    "apiKey": "API-sleutel",
+    "backToLogin": "Terug naar inloggen",
+    "beginCreateFirstUser": "Om te beginnen maakt u uw eerste gebruiker aan.",
+    "changePassword": "Wachtwoord wijzigen",
+    "checkYourEmailForPasswordReset": "Controleer uw e-mail voor een link waarmee u uw wachtwoord veilig opnieuw kunt instellen.",
+    "confirmGeneration": "Generatie bevestigen",
+    "confirmPassword": "Wachtwoord bevestigen",
+    "createFirstUser": "Eerste gebruiker aanmaken",
+    "emailNotValid": "Het ingevoerde e-mailadres is niet geldig",
+    "emailSent": "E-mail verzonden",
+    "enableAPIKey": "API-sleutel activeren",
+    "failedToUnlock": "Ontgrendeling mislukt",
+    "forceUnlock": "Forceer ontgrendeling",
+    "forgotPassword": "Wachtwoord vergeten",
+    "forgotPasswordEmailInstructions": "Vul hieronder uw e-mailadres in. U ontvangt een e-mailbericht met instructies om uw wachtwoord opnieuw in te stellen.",
+    "forgotPasswordQuestion": "Wachtwoord vergeten?",
+    "generate": "Genereren",
+    "generateNewAPIKey": "Genereer nieuwe API-sleutel",
+    "generatingNewAPIKeyWillInvalidate": "Het genereren van een nieuwe API-sleutel maakt de vorige sleutel <1>ongeldig</1>. Bent u zeker dat u wilt doorgaan?",
```

Comment:
I think that "Weet u zeker dat u wilt doorgaan" makes more sense (instead of "Bent u zeker dat u wilt doorgaan")